README.md

```diff
@@ -1,6 +1,6 @@
 # WEC-Sim_Applications
 
-[![master build status](https://github.com/WEC-Sim/WEC-Sim_Applications/actions/workflows/run-tests-master.yml/badge.svg)](https://github.com/WEC-Sim/WEC-Sim_Applications/actions/workflows/run-tests-master.yml)
+[![main build status](https://github.com/WEC-Sim/WEC-Sim_Applications/actions/workflows/run-tests-main.yml/badge.svg)](https://github.com/WEC-Sim/WEC-Sim_Applications/actions/workflows/run-tests-main.yml)
```

Comment:
@akeeste, you either need to rename the workflow file to `run-tests-main.yml` or leave the links to the files unaltered. Same goes in the Applications directory.